contracts/netmap/contract.go

```diff
@@ -442,6 +450,8 @@ func NewEpoch(epochNum int) {
 	// make clean up routines in other contracts
 	cleanup(ctx, epochNum)
 
+	storage.Put(ctx, append([]byte{epochIndexKey}, fourBytesBE(epochNum)...), curIndex)
```

Comment:
How about storing the time in the same index? We don't have use cases for it right now, but it's not hard to add and it can be useful in future if we're to try doing something time-dependent.

Reply:
Yes, structure is easier to handle and can be extended in future if need be.

Reply:
done